src/styles/App.css

```diff
@@ -1,28 +1,29 @@
-.App {
-  text-align: center;
+body {
+  background: #404040;
+  color: #f8f8f8;
+  font: 500 20px/26px 'Helvetica Neue', Helvetica, Arial, Sans-serif;
+  margin: 0;
+  padding: 0;
+  -webkit-font-smoothing: antialiased;
```

Comment:
Since we have Autoprefixer in the build pipeline, perhaps the `-webkit-` prefix here is unnecessary?